addons/project_todo/tests/test_mail_activity_todo_create.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.project.tests.test_project_base import TestProjectCommon
+import datetime
+from markupsafe import Markup
+
+class TestMailActivityTodo(TestProjectCommon):
```

Comment:
I don't know if it makes much of a difference, but you could inherit `TransactionCase` directly

Reply:
Yup works as well and will be lighter